tests/jballerina-unit-test/src/test/resources/test-src/types/tuples/tuple_negative_test.bal

```diff
@@ -50,12 +50,6 @@ function testTupleToArrayAssignmentNegative1() returns string[] {
     return y;
 }
 
-function testTupleToArrayAssignmentNegative2() returns string[] {
-    [string, string] x = ["a", "b"];
-    string[3] y = x;
```

Comment:
This is a valid negative case, why it is removed?

Reply:
I was under the impression that since `string[] y = x;`, this should also be allowed (with fillers). fixed in cacae9a